Version 8 cannot be build against net8.0

Version 8 does not support net8.0 anymore. I need to target net8.0 to be able to run unit tests. However when my ptoject targets also net8.0 I cannot upgrade library to version 8.

If it helps, with 8.0.4 I see this at the developer command prompt:
```
PS > dotnet build .\DivisiBill\DivisiBill.csproj -c:Debug -f:net8.0
...\DivisiBill.csproj : warning NU1608: Detected package version outside of dependency constraint: Xamarin.AndroidX.Lifecycle.Process 2.7.0.1 requires Xamarin.AndroidX.Lifecycle.Runtime (>= 2.7.0.1 && < 2.7.1) but version Xamarin.AndroidX.Lifecycle.Runtime 2.8.3.1 was resolved.
...\DivisiBill.csproj : error NU1202:
  Package Plugin.InAppBilling 8.0.4 is not compatible with net8.0 (.NETCoreApp,Version=v8.0). Package Plugin.InAppBilling 8.0.4 supports:
    - net8.0-android34.0 (.NETCoreApp,Version=v8.0)
    - net8.0-ios18.0 (.NETCoreApp,Version=v8.0)
    - net8.0-maccatalyst18.0 (.NETCoreApp,Version=v8.0)
    - net8.0-macos15.0 (.NETCoreApp,Version=v8.0)
    - net8.0-windows10.0.19041 (.NETCoreApp,Version=v8.0)
```
With 7.1.3 it builds just fine.

--- a/src/Plugin.InAppBilling/CrossInAppBilling.shared.cs
+++ b/src/Plugin.InAppBilling/CrossInAppBilling.shared.cs
@@ -31,7 +31,11 @@ namespace Plugin.InAppBilling
             }
         }
 
+#if ANDROID || IOS || MACCATALYST || MACOS || WINDOWS
         static IInAppBilling CreateInAppBilling() => new InAppBillingImplementation();
+#else
+        static IInAppBilling CreateInAppBilling() => null;
+#endif
 
         internal static Exception NotImplementedInReferenceAssembly() =>
 			new NotImplementedException("This functionality is not implemented in the portable version of this assembly.  You should reference the NuGet package from your main application project in order to reference the platform-specific implementation.");

--- a/src/Plugin.InAppBilling/Plugin.InAppBilling.csproj
+++ b/src/Plugin.InAppBilling/Plugin.InAppBilling.csproj
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net8.0-android;net8.0-ios;net8.0-maccatalyst;net8.0-macos</TargetFrameworks>
+    <TargetFrameworks>net8.0;net8.0-android;net8.0-ios;net8.0-maccatalyst;net8.0-macos</TargetFrameworks>
     <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net8.0-windows10.0.19041.0</TargetFrameworks>
     <LangVersion>latest</LangVersion>
-    <UseMaui>true</UseMaui>
-    <UseMauiEssentials>true</UseMauiEssentials>
+    <UseMaui Condition="'$(TargetFramework)' != 'net8.0'">true</UseMaui>
+    <UseMauiEssentials Condition="'$(TargetFramework)' != 'net8.0'">true</UseMauiEssentials>
     <SingleProject>true</SingleProject>
     <AssemblyName>Plugin.InAppBilling</AssemblyName>
     <RootNamespace>Plugin.InAppBilling</RootNamespace>

